assets/datePicker.css

```diff
@@ -0,0 +1,117 @@
+/* purgecss start ignore */
```

Comment:
I added a task to convert this css file to css module

Reply:
@ufkaya What's the reason you can not use it the same way as in the listings project? https://github.com/carforyou/carforyou-listings-web/blob/master/components/forms/datePicker.module.css

Reply:
tailwind style was not processed, and I didnt want to spend too much time now to fix it, but if you know how to solve the issue then I can change it again otherwise we can do it later, I added a task as tech debt